packages/js/product-editor/src/blocks/index.ts

```diff
@@ -1,6 +1,6 @@
 export { init as initCatalogVisibility } from './product-fields/catalog-visibility';
 export { init as initCustomFields } from './product-fields/custom-fields';
-export { init as initCustomFieldsToogle } from './product-fields/custom-fields-toggle';
+export { init as initCustomFieldsToggle } from './product-fields/custom-fields-toggle';
```

Comment:
So this one worries me a bit. If I'm following the code correctly here, this function is being exported right now as `initCustomFieldsToogle`, which means other places may be importing that using that name. In doing a search on the codebase, I'm not seeing any places that use that but there may be an area I'm overlooking.

I think this is fine to move to the next step but would love the Woo team to chime in on if this is safe to update or not

Reply:
@dkotter I thought the same way and also was not able to find any occurrences of the old variable name being used.

Reply:
I couldn't find any usage of `initCustomFieldsToogle`, so I think it's ok to rename this 👍